### ISSUE TITLE
Feedback gathered from colleagues

### ISSUE THREAD

**Issue by rickschubert** [trayio/mock-inspect#12]
The following points have been raised when the tool was being presented:

- one method uses the word "Network", the other one doesn't - the names `expectRequestMadeMatching` and `expectNetworkRequestToHaveBeenMade` are inconsistent
- The examples could be more accessible

**Comment by thomaschaplin** [comment]
> one method uses the word "Network", the other one doesn't - the names `expectRequestMadeMatching` and `expectNetworkRequestToHaveBeenMade` are inconsistent

I agree the inconsistency in function names should be resolved, I'd be happy to pick this up.

**Comment by thomaschaplin** [comment]
@rickschubert I think it would be best if we split this out into two issues. Let's also discuss if we should go ahead with the second issue or close it based on not wanting to change architectural design.

**Comment by rickschubert** [comment]
@thomaschaplin I am fine to strip out `network` as you already did in PR #13 , thanks.

Feel free to create another issue for improving the documentation, absolutely.

In regards to the last point: I would suggest that this is a `wontfix` - having an object oriented API is a design decision about this tool and is - at least in my point of view - what makes mock-inspect easy to read and use.

**Comment by thomaschaplin** [comment]
Closing this issue as it's been fixed
